### PR TITLE
Message to the user when the backend throws an error while fetching exceptions

### LINKED ISSUE #6415: Loading... shown forever after too many points error
Replay with comments:

https://app.replay.io/recording/loading-shows-forever--ba9afeef-d8cb-4ea0-a87d-898d666a1818?point=36346078019776158927135764142620818&time=15149.391304347826&hasFrames=true

### PR DESCRIPTION
**Pull request by jaril**
Fix #6415.

Before: 

https://user-images.githubusercontent.com/15959269/164375800-a04bc87a-8c7d-4db4-b3e6-01b27d5c9fbd.mov

After: 

https://user-images.githubusercontent.com/15959269/164375795-17fc7f5e-6940-40ac-ab7c-f7e3e920ad7b.mov

### CODE REVIEW

**Comment by jasonLaster** [comment]
Love this!

File: src/devtools/client/debugger/src/actions/logExceptions.js [hunk 1]
```diff
@@ -4,9 +4,11 @@
 
 //
 
+import { exceptionLogpointErrorCleared } from "devtools/client/webconsole/reducers/messages";
 import { PROMISE } from "ui/setup/redux/middleware/promise";
-const { getShouldLogExceptions } = require("../reducers/pause");
+
 const { clientCommands } = require("../client/commands");
+const { getShouldLogExceptions } = require("../reducers/pause");
```
Comment (markerikson) [inline src/devtools/client/debugger/src/actions/logExceptions.js:11]: :hand: Might as well change this to be an `import` statement at this point

File: src/devtools/client/debugger/src/actions/logExceptions.js [hunk 2]
```diff
@@ -21,10 +23,12 @@ export function setupExceptions(store) {
...
 export function logExceptions(shouldLogExceptions) {
   return (dispatch, getState, { client }) => {
+    dispatch(exceptionLogpointErrorCleared());
+
     return dispatch({
-      type: "LOG_EXCEPTIONS",
-      shouldLogExceptions,
       [PROMISE]: client.logExceptions(shouldLogExceptions),
+      shouldLogExceptions,
+      type: "LOG_EXCEPTIONS",
```
Comment (markerikson) [inline src/devtools/client/debugger/src/actions/logExceptions.js:31]: :question: Is whatever new auto-sorting setup actually sorting object keys?  If so, that seems heavy-handed

File: src/protocol/logpoint.ts [hunk 3]
```diff
@@ -536,9 +538,15 @@ export async function setExceptionLogpoint(logGroupId: string) {
   try {
     await analysisManager.runAnalysis(params, handler);
   } catch (e: any) {
+    removeLogpoint(logGroupId);
+    let msg;
     if (e.code === ProtocolError.TooManyPoints) {
-      store.dispatch(onConsoleOverflow());
+      msg = "There are too many exceptions. Please focus and try again.";
```
Comment (markerikson) [inline src/protocol/logpoint.ts:544]: :hand: Might be worth making this message a bit longer, like "Please focus to a smaller time range and try again".

File: src/ui/components/UploadScreen/TeamSelect.tsx [hunk 4]
```diff
@@ -36,7 +36,7 @@ export default function TeamSelect({
   let displayedWorkspaces: DisplayedWorkspace[] = [...workspaces].sort();
 
   if (userInfo.features.library) {
-    displayedWorkspaces = [personalWorkspace, ...displayedWorkspaces];
+    displayedWorkspaces = [personalWorkspace];
```
Comment (markerikson) [inline src/ui/components/UploadScreen/TeamSelect.tsx:39]: :question: What's the change here in relation to the rest?

**Review by markerikson — APPROVED** [review]
Couple questions and suggested cleanup nits, but seems okay enough overall

File: src/ui/components/UploadScreen/TeamSelect.tsx (hunk 4 above)
Comment (jaril) [reply src/ui/components/UploadScreen/TeamSelect.tsx:39]: GOOD CALL, this was my test code. This is why reviews are nice :)

File: src/devtools/client/debugger/src/actions/logExceptions.js (hunk 2 above)
Comment (jaril) [reply src/devtools/client/debugger/src/actions/logExceptions.js:31]: It is!

**Pull request merged**